src/tests/topp/QCCalculator_2_output.mzQC

```diff
@@ -71,7 +71,7 @@
             "name": "MZ acquisition range",
             "value": [
               0,
-              445
+              18
```

Comment:
now the MZ acquisition range is small for that data.
@cbielow if you want to include precursor ranges you could do this here